### ISSUE TITLE
Duplicated Armour Stands in structures

### ISSUE THREAD

**Issue by BrandoniRigatoni** [jtorleon-studios-team/bettervillages#107]
Hi, I have this mod in a mod-pack i downloaded, There are no other 'custom structure' mods in it. Some buildings have armor stands with a full iron set. When i go to break the stand like i would a boat, the armor on it along with the stand itself drops on the ground while the stand is still physically there. I can duplicate it four times before the stand breaks. Its not game-changing, as its only iron armor, but it also clogs my inventory when I am trying to renovate which is annoying. Does anybody else have this issue and if so, a fix?  I play on MC Version 1.20.1  The Mod version is Forge 1.20.1-3.2.0

**Comment by BrandoniRigatoni** [comment]
I should also add that this is the Forge version 1.20.1-3.2.0

**Comment by Metysm** [comment]
Hello, I have the same issue on 1.19.2, do you know from which mod your structures was?

**Comment by Hqrvester** [comment]
Also same issue with some item frames

**Comment by TheShadyyOneYT** [comment]
have the same issue to

**Comment by vinnyg0621** [comment]
> have the same issue to, would be great to have this fixed :)

have the same issue to,…. you didn’t finish that part.

**Comment by TheShadyyOneYT** [comment]
What do you mean? What am I missing?  What part have I not finished? By the way this isn't just a forge issue, its also on fabric as well, same mod version. It's both modloaders fabric and forge, but the only difference is that the item frames don't duplicate on fabric. Only the armor stands, 4 stacked on each other.

**Comment by vinnyg0621** [comment]
issue to what, who, or where?

**Comment by TheShadyyOneYT** [comment]
There is no who, wtf are you talking about. How do you not understand the issue, the armor stands in the houses, are stacked by 4 times, if you break an armor stand with armor, it will have 3 more sets stacked on top of it, test it for yourself. I already made a separate issue for fabric, fabric is a modloader, and so is forge. How hard is to understand the main problem which is @BrandoniRigatoni stated. Help me understand what you don't get. I'm honestly confused.

**Comment by vinnyg0621** [comment]
i have the same issue too.
**wait**! @TheShadyyOneYT is that what you meant? “i have the same issue too”? because i have the same issue, and it would be great to have fixed! 😃

**Comment by TheShadyyOneYT** [comment]
ok good to know you're having the issue too. its been bugging me recently.

**Comment by vinnyg0621** [comment]
yeah i didn't notice it when i was looking at the structure files until I broke one of 'em.

**Comment by TheShadyyOneYT** [comment]
I was playtesting my mdpack using this mod and just realized the armor stand had 4 stacked upon each other and my initial reaction was just "bruh". because now we have to wait like 3 months for this to be fixed ;-;

**Comment by vinnyg0621** [comment]
> I was playtesting my mdpack using this mod and just realized the armor stand had 4 stacked upon each other and my initial reaction was just "bruh". because now we have to wait like 3 months for this to be fixed ;-;

3 months for it to be fixed? how'd you know when the next mod update comes? well, I'm hyped for November then!! 😃

**Comment by TheShadyyOneYT** [comment]
> > I was playtesting my mdpack using this mod and just realized the armor stand had 4 stacked upon each other and my initial reaction was just "bruh". because now we have to wait like 3 months for this to be fixed ;-;
> 
> 3 months for it to be fixed? how'd you know when the next mod update comes? well, I'm hyped for November then!! 😃

I don’t know when it comes out. I was just stating mod updates usually aren’t a quick hot fix. They usually can range from weeks to months for things to be updated. Not to mention back porting can take even longer. I’m no expert at all, but I said “like 3 months” because I’m hoping it won’t take a full year for an update but at same time I don’t expect it in a week.

**Comment by jtl-elisa** [comment]
Done.
Will be available in the next update **3.2.2**
